api/v1alpha1/webhook_suite_test.go

```
@@ -0,0 +1,96 @@
+	go func() {
+		if err := mgr.Start(ctx); err != nil {
+			t.Logf("manager exited: %v", err)
+		}
+	}()
+
+	// Wait for webhook server to accept TLS connections
+	dialer := &net.Dialer{Timeout: time.Second}
+	addrPort := fmt.Sprintf("%s:%d", webhookOpts.LocalServingHost, webhookOpts.LocalServingPort)
+
+	require.Eventually(t, func() bool {
+		conn, err := tls.DialWithDialer(dialer, "tcp", addrPort, &tls.Config{InsecureSkipVerify: true})
```

golangci-lint (2.11.4)

[error] crypto/tls.DialWithDialer must not be called. use (*crypto/tls.Dialer).DialContext with NetDialer

(noctx)